Packages/DesignSystem/Sources/DesignSystem/Views/ThemePreviewView.swift

```diff
@@ -0,0 +1,109 @@
+import SwiftUI
+import Combine
+
+public struct ThemePreviewView: View {
+  private let gutterSpace: Double = 8
+  @EnvironmentObject private var theme: Theme
+  @Environment(\.dismiss) var dismiss
+  
+  public init() {}
+  
+  public var body: some View {
+    VStack {
+      HStack {
+        Text("Select Theme")
```

Comment:
I would be in favour of using a NavigationStack with a title and a toolbar item as a close button. Only because it make it consistent with all other screens in the app. Unless this is hard push for your own style there (which is fine too) :)

Reply:
I agree. Will update.